.ddev/config.yaml

```diff
@@ -18,6 +18,7 @@ hooks:
         [[ -d frontend ]] || git clone ${FRONTEND_REPOSITORY:-https://github.com/drunomics/lupus-decoupled-nuxt3-demo} --branch=${FRONTEND_BRANCH:-main} frontend
         cd frontend && npm i
         set -x
+        npm run init-project-files
```

Comment:
If we use the shadcn demo, this post start hook will fail. We should find a way to move this command into the FE repo itself.